app/src/main/res/values-night/themes.xml

```diff
@@ -10,7 +10,7 @@
         <item name="colorSecondaryVariant">@color/appyx_dark</item>
         <item name="colorOnSecondary">@color/appyx_bright</item>
         <!-- Status bar color. -->
-<!--        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>-->
+<!--        <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>-->
```

Comment:
This causes a lint problem (in the uncommented version). I decided to change the commented out ones as well just in case we use them in the future.